src/mono/wasm/Wasm.Build.Tests/data/WasmOverridePacks.targets

```diff
@@ -0,0 +1,63 @@
+<Project>
```

Comment:
Everything in this file is a copy of targetingpacks.targets. Why is that?

Reply:
This was added in the wasm.build.tests quite some time ago, and then removed. I can try using `targetingpacks.targets`. I can make a copy of it at build time, to use with test project projects that are generated, and built for individual tests.

Reply:
Not required for this PR but it would be good to file a follow-up issue to remove the duplication.

Reply:
https://github.com/dotnet/runtime/issues/91350